bot/persistence/database_connector.py

```diff
@@ -269,6 +261,69 @@ def is_reaction_role_uniqueness_group(self, msg_id: int) -> bool:
 
             return bool(row[0])
 
+    def get_translation(self, msg_id: int, flag_emoji: str) -> Optional[str]:
+        """Gets the translation for the specified message.
+
+        Args:
+            msg_id (int): The id of the message which translation is needed.
+            flag_emoji (str): The emoji representing the language.
+
+        Returns:
+            Optional[str]: The translation of the message.
+        """
+        with DatabaseManager(self._db_file) as db_manager:
+            result = db_manager.execute(queries.GET_TRANSLATION, (msg_id, flag_emoji))
+
+            row = result.fetchone()
+            return row[0]
```

Comment:
Can row be ` None` in this case? If so, potential runtime error, bc you try to get the first element of `None`

Reply:
Yep, you're right. I gonna fix this.